hashedixsearch/search.py

```diff
@@ -149,8 +167,15 @@ def highlight(query, terms, stemmer, analyzer):
 
     # Generate unstemmed ngrams of the maximum term length
     ngrams = []
-    for tokens in tokenize(doc=query, ngrams=max_n, analyzer=analyzer):
-        if len(tokens) < max_n:
+    for tokens in tokenize(
+        doc=query,
+        ngrams=(2 * max_n) - 1,
```

Comment:
Similar to https://github.com/openculinary/hashedixsearch/pull/2/files#r418558043, this multiplication is only required because we have uncertainty about the length of the `terms` relative to the length of `ngrams`.  It would be an improvement to revert this to a simple `ngrams=max_n` argument.